--- lib.rs	2020-06-12 20:43:54.000000000 -0400
+++ lib.new.rs	2020-06-16 17:35:21.331484300 -0400
@@ -33,7 +33,7 @@
 
 pub use crate::config::{
     load_config, CliOptions, Color, Config, Edition, EmitMode, FileLines, FileName, NewlineStyle,
-    Range, Verbosity,
+    Range, Verbosity, PartialConfig
 };
 
 pub use crate::format_report_formatter::{FormatReportFormatter, FormatReportFormatterBuilder};
